src/client/resources/web/js/messages/message_parsing.mjs

```diff
@@ -936,6 +966,143 @@ function formatHoverEvent(hoverEvent, translations) {
+
+                const closeModal = () => {
+                    modalContainer.style.display = 'none';
+                    modalUrlElement.textContent = '';
+                    modalConfirmButton.removeEventListener(
+                        'click',
+                        confirmHandler,
+                    );
+                    modalCancelButton.removeEventListener(
+                        'click',
+                        cancelHandler,
+                    );
+                    modalCopyButton.removeEventListener('click', copyHandler);
+                    modalConfirmButton.removeEventListener(
+                        'click',
+                        confirmHandler,
+                    );
+                    modalContainer.removeEventListener('click', closeModal);
+                    modalContent.removeEventListener(
+                        'click',
+                        contentClickHandler,
+                    );
+                };
+
+                const contentClickHandler = (
+                    /** @type {MouseEvent} */ event,
+                ) => {
+                    event.stopPropagation();
+                };
+
+                const cancelHandler = () => {
+                    modalUrlElement.textContent = '';
+                    closeModal();
+                };
+
+                const copyHandler = () => {
+                    navigator.clipboard.writeText(clickEvent.value);
+                    closeModal();
+                };
+
+                const confirmHandler = () => {
+                    window.open(
+                        clickEvent.value,
+                        '_blank',
+                        'noopener,noreferrer',
+                    );
+                    closeModal();
+                };
+
+                modalCancelButton.addEventListener('click', cancelHandler);
+                modalCopyButton.addEventListener('click', copyHandler);
+                modalConfirmButton.addEventListener('click', confirmHandler);
+                modalContainer.addEventListener('click', closeModal);
+                modalContent.addEventListener('click', contentClickHandler);
+                modalContainer.style.display = 'block';
+            };
+        case 'suggest_command':
+            return (event) => {
+                if (event.shiftKey) {
+                    return;
+                }
+
+                const chatInputElement = /** @type {HTMLTextAreaElement} */ (
+                    querySelectorWithAssertion('#message-input')
+                );
+                chatInputElement.value = clickEvent.value;
+                chatInputElement.focus();
+            };
+        case 'copy_to_clipboard':
+            return (event) => {
+                if (event.shiftKey) {
+                    return;
+                }
+
+                navigator.clipboard.writeText(clickEvent.value);
+            };
+        case 'run_command':
+            return (event) => {
```

Comment:
Because of the new client-side validation of supported commands I'm comfortable adding support for this. On the off-chance the command is one that we support this will be the ideal user experience. If it's not, the user should generally understand what went wrong.